src/index.html

```diff
@@ -7,8 +7,23 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap&subset=cyrillic" rel="stylesheet">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <img class="logo" src="images/logo.png" alt="logo">
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item"><a class="nav__link is-active" href="">APPLE</a></li>
+        <li class="nav__item"><a class="nav__link" href="">SAMSUNG</a></li>
+        <li class="nav__item"><a class="nav__link" href="">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+        <li class="nav__item" data-qa="hover"><a class="nav__link" href="">НОУТБУКИ И КОМПЬЮТЕРЫ</a></li>
+        <li class="nav__item"><a class="nav__link" href="">ГАДЖЕТЫ</a></li>
+        <li class="nav__item"><a class="nav__link" href="">ПЛАНШЕТЫ</a></li>
+        <li class="nav__item"><a class="nav__link" href="">ФОТО</a></li>
+        <li class="nav__item"><a class="nav__link" href="">ВИДЕО</a></li>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.